### PR TITLE
Clean up katello-selinux of foreman entities

### PR DESCRIPTION
**Pull request by chris1984**
@lzap @ekohl let me know how this looks.

### CODE REVIEW

File: katello-selinux-relabel [hunk 1]
```diff
@@ -1,26 +1,6 @@
 #!/bin/bash
 
-# relabel foreman
```
Comment (ekohl) [inline katello-selinux-relabel]: Should it call `foreman-selinux-relabel`?

Comment (lzap) [reply katello-selinux-relabel]: Hmmm you are right, let me brainstorm here. As a user, you want to have everything to be relabeled if you do:

    # foreman-selinux-relabel

Therefore we should perhaps do the opposite - to call this script from foreman's one. Now, we don't want to create dependency like that, we should perhaps go with either some relabel.d/ directory mechanism, or simply do call all scripts which match this pattern: /usr/sbin/*-selinux-relabel. Now that's a bad idea, I think we need to come up with relabel.d/ or similar approach.

Alternatively, we can leave this as a dummy script that will fail with "Please use foreman-selinux-relabel" and move the line there. Opinions?

Comment (chris1984) [reply katello-selinux-relabel]: I am for the easier option, @ekohl what do you suggest?

Comment (lzap) [reply katello-selinux-relabel]: Calling scripts from .d directory is quite easy... It's just packaging work there: https://github.com/theforeman/foreman/blob/develop/script/foreman-debug#L375-L385

**Comment by lzap** [comment]
Hey, this should solve this. You just delete the file and drop an enable script:

https://github.com/theforeman/foreman-selinux/pull/79

**Comment by lzap** [comment]
@chris1984 the referenced PR won't do the trick, you need to use hooks to do what you want.

**Pull request closed without merging**